.github/workflows/build.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.17
+          go-version: 1.18
```

Comment:
You don't need to change anything but I wanted to show you a nice improvement to this gh action in their latest v3 release that uses the go version defined in your go.mod and it automatically sets up caching too

```yaml
- uses: actions/setup-go@v3
   with:
     go-version-file: go.mod
     cache: true
     cache-dependency-path: go.sum
```